frameworks/hdfs/tests/test_sanity.py

```diff
@@ -143,10 +146,11 @@ def test_kill_all_journalnodes():
     journal_ids = sdk_tasks.get_task_ids(FOLDERED_SERVICE_NAME, 'journal')
     data_ids = sdk_tasks.get_task_ids(FOLDERED_SERVICE_NAME, 'data')
 
-    for host in shakedown.get_service_ips(FOLDERED_SERVICE_NAME):
-        sdk_tasks.kill_task_with_pattern('journalnode', host)
+    for journal_pod in get_pod_type_instances("journal", FOLDERED_SERVICE_NAME):
+        sdk_cmd.run_cli('hdfs --name={} pod restart {}'.format(FOLDERED_SERVICE_NAME, journal_pod))
+        # wait for the entire service to recover as name nodes will crash if quorum is lost among journal nodes
+        expect_recovery(service_name=FOLDERED_SERVICE_NAME)
```

Comment:
Leave this outside the loop.  Let the name nodes crash.  It's fine.  The service should recover.